### PR TITLE
EIP155 commit

### PR DESCRIPTION
**Pull request by 516108736**

### CODE REVIEW

File: cluster/slave/api_backend.go [hunk 1]
```diff
@@ -207,7 +187,7 @@ func (s *SlaveBackend) GetAccountData(address *account.Address, height *uint64)
 		}
 		hash, err := shard.MinorBlockChain.GetHashByHeight(height)
 		if err != nil {
-			return nil, err
+			continue
```
Comment (ping-ke) [inline cluster/slave/api_backend.go:190]: why?

File: cluster/slave/backend.go [hunk 2]
```diff
@@ -53,6 +53,9 @@ func (s *SlaveBackend) GetFullShardList() []uint32 {
 	return s.fullShardList
 }
 
+func (s *SlaveBackend) GetClusterConfig() *config.ClusterConfig {
+	return s.clstrCfg
+}
```
Comment (ping-ke) [inline cluster/slave/backend.go:58]: add empty line

File: cluster/slave/filters/filter_system.go [hunk 3]
```diff
@@ -7,6 +7,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/QuarkChain/goquarkchain/cluster/config"
+
```
Comment (ping-ke) [inline cluster/slave/filters/filter_system.go:11]: remove empty line

File: internal/qkcapi/api.go [hunk 4]
```diff
@@ -64,31 +61,64 @@ func (c *CommonAPI) SendRawTransaction(encodedTx hexutil.Bytes) (hexutil.Bytes,
...
+	}
+	header := block.Header()
+	tx := block.Transactions()[i]
+	from, err := tx.Sender(types.NewEIP155Signer(clusterCfg.Quarkchain.NetworkID))
 	if err != nil {
 		return nil, err
 	}
-	branch := account.Branch{Value: fullShardId}
-	minorBlock, index, receipt, err := c.b.GetTransactionReceipt(txHash, branch)
+	field := map[string]interface{}{
```
Comment (ping-ke) [inline internal/qkcapi/api.go:85]: will this change impact the current service, does python need to change also?

File: internal/qkcapi/api.go [hunk 5]
```diff
@@ -348,7 +378,7 @@ func (p *PublicBlockChainAPI) GetMinorBlockByHeight(fullShardKey hexutil.Uint, h
 		return nil, err
 	}
 	if needExtraInfo == nil {
-		temp := true
+		temp := false
```
Comment (ping-ke) [inline internal/qkcapi/api.go:381]: why?

Comment (516108736) [reply internal/qkcapi/api.go:381]: will revert

File: cluster/slave/backend.go (hunk 2 above)
Comment (516108736) [reply cluster/slave/backend.go]: will remove

File: internal/qkcapi/api.go (hunk 4 above)
Comment (516108736) [reply internal/qkcapi/api.go]: will revert

**Review by ping-ke — APPROVED** [review]

**Pull request merged**